src/mqtt.rs

```diff
@@ -13,14 +15,221 @@ pub struct Mqtt {
     client: Client,
 }
 
+
+/// `HMSStateResponse` is a struct that contains the data from the inverter.
+///
+/// Provide utility functions to extract data from the struct.
+impl HMSStateResponse {
+    fn get_model(&self) -> String {
+         // TODO: identify model from dtu_sn
+        "HMS-800W-T2".to_string()
```

Comment:
This is not necessarily the inverter. While tested, the tool likely works with all devices of the HMS-XXXX-2T series.

Reply:
Right, since we currently don't have a way to determine the exact model name, I've removed the mention of 800W-2T